### PR TITLE
Make user model swappable

### PR DESCRIPTION
**Pull request by emaciupe**

### CODE REVIEW

File: src/unicef_security/models.py [hunk 1]
```diff
@@ -8,12 +8,15 @@
 app_label = 'unicef_security'
 
 
-class TimeStampedModel:
-    last_modify_date = models.DateTimeField(editable=False, blank=True, auto_now_add=True,
-                                            auto_now=True)
+class TimeStampedModel(models.Model):
```
Comment (domdinicola) [inline src/unicef_security/models.py:11]: I guess we can import this from model_utils

File: src/unicef_security/models.py [hunk 2]
```diff
@@ -47,15 +50,17 @@ class Meta:
         swappable = 'BUSINESSAREA_MODEL'
 
 
-class User(AbstractUser, TimeStampedModel):
+class ExtendedAbstractUser(AbstractUser, TimeStampedModel):
```
Comment (domdinicola) [inline src/unicef_security/models.py:53]: can't this be the swappable model?

Comment (emaciupe) [reply src/unicef_security/models.py:53]: To my knowledge, you cannot have both abstract and swappable

File: src/unicef_security/models.py (hunk 1 above)
Comment (emaciupe) [reply src/unicef_security/models.py:11]: sure, not sure how much it matters, but using model_utils will remove `last_modify_date` field from olc user

File: src/unicef_security/models.py (hunk 2 above)
Comment (domdinicola) [reply src/unicef_security/models.py:53]: so swappable should be fine, no?

Comment (emaciupe) [reply src/unicef_security/models.py:53]: yes :) since we're not going to use it on olc

File: src/unicef_security/models.py [hunk 3]
```diff
@@ -66,10 +71,15 @@ def label(self):
...
 
     def save(self, *args, **kwargs):
         if not self.display_name:
             self.display_name = self.label
         super().save(*args, **kwargs)
+
+
+class User(ExtendedAbstractUser):
+    class Meta:
+        swappable = 'AUTH_USER_MODEL'
```
Comment (robertavram) [inline src/unicef_security/models.py:84]: why are we doing this?

Comment (domdinicola) [reply src/unicef_security/models.py:84]: hey Rob I own you this!
Swappable is when you can define a model as setting, (User model in django is swappable). FK would go to the user model, this should not have a big impact.
My idea was to have a User model swappable/mixin that can be applied to our applications and we can share common functionalities (AD integration, AD login, pipelines..) and fields (azure_id, job_title.. )

**Pull request closed without merging**